src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -186,6 +196,10 @@ class BasePaymentsPage extends React.Component {
         throw new Error('Invalid payment method type selected');
     }
 
+    fetchData() {
+        PaymentMethods.getPaymentMethods();
```

Comment:
Why are we wrapping this function in `fetchData` instead of just calling it directly like before?

Reply:
I'm still a bit unclear on this...